EpiserverRedirects/Request/RequestHandler.cs

```diff
@@ -24,9 +25,16 @@ public class RequestHandler
         public async Task Invoke(Uri request, IHttpResponse response)
         {
             var requestPath = UrlPath.FromUri(request);
-            
+
             var redirectRule = await _redirectRuleResolver.ResolveRedirectRuleAsync(requestPath);
 
+            if (redirectRule is NullRedirectRule)
```

Comment:
This check was added to implement backward compatibility with existing redirects that have unencoded spaces in the urls

Reply:
how about altering matching logic in implementations of `ResolveRedirectRuleAsync` method (ones in `ExactMatchResolver` and `RegexResolver` needs updating) , so 

`FirstOrDefault(r => r.OldPattern == oldPath.ToString())` escapes url in both sides of equation? This case no fallback is required